docs/components/concepts/process-instance-migration.md

```diff
@@ -169,6 +169,45 @@ You can use [process instance modification](./process-instance-modification.md)
 This results in new keys for the service task as well as the job.
 :::
 
+## Dealing with attached boundary events
+
+You can migrate active elements with boundary events attached.
+You decide what happens to the associated event subscription through the mapping instructions for the boundary events:
+
+- If a boundary event is mapped, the associated subscription is migrated. Please note, this case is not yet supported.
+- If a boundary event in the source process is not mapped, then the associated subscription is closed during migration.
+- If a boundary event of the target process is not the target of a mapping instruction, then a new subscription is opened during migration.
```

Comment:
🔧 Would be more explicit if we add the case where both source and target definitions has boundary events and mention closing subscription in the source and opening brand new in the target definition.

Reply:
Interesting idea. It is the combination of the 2nd and 3rd bullet points. In the end, there are only 3 cases: mapped source (also includes a target), unmapped source, and unmapped target. As C7 docs also didn't mention this explicitly, I think we can do without it. If someone feels strongly about this, or it leads to confusion then we can add it later. (Please let us know here)